eng/CodeChecks.ps1

```diff
@@ -69,12 +64,12 @@ try {
 
     Write-Host "Re-generating readmes"
     Invoke-Block {
-        & $PSScriptRoot\Update-Snippets.ps1 @script:PSBoundParameters
+        & $PSScriptRoot\Update-Snippets.ps1 $ServiceDirectory
```

Comment:
I don't recommend this change. It's already a parameter - you can update it since `$PSBoundParameters` is just a hashtable. Then still splat it, which also carries forward any `-Verbose` or other parameters.

```powershell
$PSBoundParameter['ServiceDirectory'] = $ServiceDirectory
# ...
& $PSScriptRoot\Update-Snippets.ps1 @script:PSBoundParameters
```